kolibri/plugins/learn/assets/src/views/TopicsPage/TopicsPanelModal.vue

```diff
@@ -106,14 +106,15 @@
   }
 
   .side-panel {
+    @extend %dropshadow-2dp;
```

Comment:
Same as the comment for [SearchFiltersPanel/index.vue](https://github.com/learningequality/kolibri/pull/12630/files#diff-16a6ba2aaca47edd01098e6c42e746e77f4c247dc44dd2f9ff2559b128764c21)